src/app/dashboard/events/page.tsx

```diff
@@ -1,17 +1,113 @@
+import { format } from "date-fns";
+import {
+  AlertCircle,
+  Calendar1,
+  CircleHelpIcon,
+  Share2Icon,
+  Users,
+} from "lucide-react";
+import Image from "next/image";
 import Link from "next/link";
+import { notFound } from "next/navigation";
 
+import EventPhotoPlaceholder from "@/../public/event-photo-placeholder.png";
+import { CreateEventForm } from "@/app/dashboard/(create-event)/create-event-form";
+import { EventInfoDiv } from "@/components/event-info-div";
+import { Alert, AlertDescription, AlertTitle } from "@/components/ui/alert";
 import { Button } from "@/components/ui/button";
+import { API_URL, PHOTO_URL } from "@/lib/api";
+import { verifySession } from "@/lib/session";
+import type { Event } from "@/types/event";
 
-export default function EventListPage() {
+export default async function EventListPage() {
+  const session = await verifySession();
+  if (session == null || typeof session.bearerToken !== "string") {
+    notFound();
+  }
+  const { bearerToken } = session;
+  const response = await fetch(`${API_URL}/events`, {
+    headers: {
+      Authorization: `Bearer ${bearerToken}`,
+    },
+  });
+  if (!response.ok) {
+    return (
+      <div className="flex w-full flex-col items-center gap-4">
+        <Alert variant="destructive">
+          <AlertCircle className="size-6" />
+          <AlertTitle>Wystąpił błąd podczas pobierania danych.</AlertTitle>
+          <AlertDescription>
+            Sprawdź swoje połączenie z internetem i spróbuj ponownie.
+          </AlertDescription>
+        </Alert>
+      </div>
+    );
+  }
+  const events = (await response.json()) as Event[];
   return (
     <div className="flex flex-col gap-4">
-      <h1 className="text-3xl font-bold">Wszystkie wydarzenia - 0</h1>
-      <Button className="max-w-fit" asChild>
-        {/* TODO display list of events */}
-        <Link href="/dashboard/events/2">
-          Kliknij by przejść do widoku zarządzania eventem
-        </Link>
-      </Button>
+      <div className="space-y-8">
+        <div className="flex justify-between">
```

Comment:
Nieresponsywne. Fix:
```suggestion
        <div className="flex flex-col justify-between gap-4 md:flex-row">
```